src/ROUTES.ts

```diff
@@ -585,6 +585,10 @@ const ROUTES = {
         route: 'settings/workspaces/:policyID/tags/edit',
         getRoute: (policyID: string) => `settings/workspaces/${policyID}/tags/edit` as const,
     },
+    WORKSPACE_TAG_EDIT: {
+        route: 'workspace/:policyID/tags/:tagName/edit',
```

Comment:
This should be the correct route for edit
```suggestion
        route: 'workspace/:policyID/tag/:tagName/edit',
```